### ISSUE TITLE
Empty Timeline, others don't see me following

### ISSUE THREAD

**Issue by LeoSum8** [nextcloud/social#376]
**Describe the bug**
I notice several issues, that I believe are related.
1. I am able to add profiles to my "following" list by searching for them and clicking follow in my nextcloud instance. However I do not appear in their "follower" list.
2. My global timeline stays empty. There are no entries but my own.

**To Reproduce**
1. Follow my own profile on another mastodon instance by searching the name and clicking follow
2. Check that the profile is listed under "following"
3. go to profile on other mastodon instance and check followers
4. write messages back and forth using @mentions between the profiles

**Expected behavior**
I would expect to see the following/follower status on the profiles and also the messages on the timelines

**Client details:**
 - OS: Windows/Linux
 - Browser firefox/chrome
 - Version 64.0.2/67
 - Device: desktop

<details>
<summary>Server details</summary>

**Social app version: 0.1.3**

**Operating system: FreeBSD**

**Web server: apache**

**Database:mariadb**

**PHP version:7**

**Nextcloud version:15.0.2** (see Nextcloud admin page)

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
no relevant logs
```

#### Browser log
```
no relevant logs
```

</details>

**Comment by LeoSum8** [comment]
I imagine that this is a little sparse info to diagnose antyhing. What additional info shall I provide?

**Comment by LeoSum8** [comment]
done! thanks!

**Comment by daita** [comment]
Can you make your redirect more like this (in the .htaccess at the root of your website)

```
Redirect /.well-known/webfinger https://cloud.example.net/nextcloud/.well-known/webfinger
```

Also, check the .htaccess which is at the root of your Nextcloud and see if the line about webfinger point to /nextcloud/public.php[...]:

```
  RewriteRule ^\.well-known/webfinger /nextcloud/public.php?service=webfinger [QSA,L]                                                                                                          
```

**Comment by pongraczi** [comment]
Hi, I have similar problem:  
- I cannot see any posts from my contacts on Mastodon / global list empty, no posts from them
- the private message sent from my nextcloud to them does not appear on his message board
- from Mastodon I cannot find my nextcloud account
- from my nextcloud I can search and add contacts from Mastodon (nextcloud@mastodon.xyz, pongraczi@mastodon.social), their avatars and mottos are visible
![kijeloles_714](https://user-images.githubusercontent.com/5891983/52526757-23f24880-2cbe-11e9-96bc-611983550ef6.jpg)


Details:  
- nextcloud is under a folder, not in the docroot
- I modified the .htaccess until the selftest succeed
(   RewriteRule ^\.well-known/webfinger https://%{SERVER_NAME}/cbox/public.php?service=webfinger [QSA,L] )
I got: _missing resource_ when I access to the webfinger, so, it should work
- latest nextcloud (15.0.4) and social (0.1.4)

**Comment by LeoSum8** [comment]
> Can you make your redirect more like this (in the .htaccess at the root of your website)
> 
> ```
> Redirect /.well-known/webfinger https://cloud.example.net/nextcloud/.well-known/webfinger
> ```
> 
> Also, check the .htaccess which is at the root of your Nextcloud and see if the line about webfinger point to /nextcloud/public.php[...]:
> 
> ```
>   RewriteRule ^\.well-known/webfinger /nextcloud/public.php?service=webfinger [QSA,L]                                                                                                          
> ```

So you suggest sort of daisy-chaining the webfinger redirects, do I understand you correctly?

My current settings for webfinger are as follows:

.htaccess in website root:
```
RewriteRule ^\.well-known/webfinger https://cloud.example.net/nextcloud/public.php?service=webfinger [QSA,L]
```

.htaccess in nextcloud subfolder:
```
RewriteRule ^\.well-known/webfinger public.php?service=webfinger [QSA,L]
```

with this setup I get no warnings in the nextcloud settings page, and I can call the well-known webfinger both via
```
https://cloud.example.net/nextcloud/.well-known/webfinger
```
and
```
https://cloud.example.net/.well-known/webfinger
```

After your suggestion, I changed that to the following:

.htaccess in website root:
```
RewriteRule ^\.well-known/webfinger https://cloud.example.net/nextcloud/.well-known/webfinger [QSA,L]
```

.htaccess in nextcloud subfolder:
```
RewriteRule ^\.well-known/webfinger /nextcloud/public.php?service=webfinger [QSA,L]
```

After this, I cannot call the webfinger via `https://cloud.example.net/nextcloud/.well-known/webfinger`, nor via `https://cloud.example.net/.well-known/webfinger`. Also the check in settings fails and tells me something is wrong with my webfinger setup.

However if I change the .htaccess in nextcloud subfolder to:
```
RewriteRule ^\.well-known/webfinger public.php?service=webfinger [QSA,L]
```
I can call the webfinger again as before and the check in settings works again with the daisy chain in place.
However all this didn't change the initial problem with the empty timeline.

Thank you again for looking into this!

**Comment by Glebelg2** [comment]
Dear, I face exactly the same problem as pongraczi.
 
I can follow mastodon accounts but I can't see any messages (nor when clicking on their profile nor in the global timeline).

Nextcloud 15.0.4
Social 0.1.4
Apache 2.4.38

my .htaccess shows:
`...
  RewriteRule ^\.well-known/webfinger https://my.server/public.php?service=webfinger [QSA,L]
...`


Thanks for your help

**Comment by rikmeijer** [comment]
This pull request helped fixing the empty timeline https://github.com/nextcloud/social/pull/490

**Comment by daita** [comment]
is it fixed in 0.2.6 ?

**Comment by LeoSum8** [comment]
Yes, although I'm not sure if it was fixed by 0.2.6 or by running the `occ social:reset` command.
Sorry!

**Comment by pongraczi** [comment]
Yes, it seems working well, there are posts as expected:

![Kijelölés_074](https://user-images.githubusercontent.com/5891983/60669351-68010800-9e6e-11e9-9529-1f0562fe7144.png)
